src/entry/Entry.tsx

```diff
@@ -162,6 +164,12 @@ const Entry: React.FC<EntryProps> = ({
       ),
       mainContent: <EntryMain transformedData={entryData} />,
     },
+    {
+      name: 'Feature viewer',
+      path: 'feature-viewer',
+      icon: <ProteinIcon />,
```

Comment:
Using the protein icon for now (same one than the loading component), awaiting better one from Austra